tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -983,6 +1002,100 @@ public function testCreateDocument(array $data): array
         return $data;
     }
 
+
+    /**
+     * @depends testCreateDocument
+     */
+    public function testTimeouts(array $data): void
+    {
+        $documents = [];
+        for ($i = 0; $i <= 1; $i++) {
+            $documents[] = $this->client->call(Client::METHOD_POST, '/databases/' . $data['databaseId'] . '/collections/' . $data['moviesId'] . '/documents', array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+            ], $this->getHeaders()), [
+                'documentId' => ID::unique(),
+                'data' => [
+                    'title' => 'title',
+                    'releaseYear' => 2020,
+                    'longtext' => file_get_contents(__DIR__ . '/longtext'),
+                ],
+                'permissions' => [
+                    Permission::read(Role::user($this->getUser()['$id'])),
+                    Permission::update(Role::user($this->getUser()['$id'])),
+                    Permission::delete(Role::user($this->getUser()['$id'])),
+                ]
+            ]);
+        }
+
+        $docs = [];
+        for ($i = 0; $i <= 2; $i++) {
+            $docs[] = $this->client->call(Client::METHOD_GET, '/databases/' . $data['databaseId'] . '/collections/' . $data['moviesId'] . '/documents', array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+                'x-appwrite-timeout' => 1,
+            ], $this->getHeaders()), [
+                //'queries' => ['sleep("$id", 1)'],
+                'queries' => ['notEqual("longtext", "appwrite")'],
+            ]);
+        }
+
+        for ($i = 0; $i < count($documents); $i++) {
+            $this->client->call(Client::METHOD_DELETE, '/databases/' . $data['databaseId'] . '/collections/' . $data['moviesId'] . '/documents/' . $documents[$i]['body']['$id'], array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+            ], $this->getHeaders()));
+        }
+
+        $this->assertEquals(408, $docs[0]['headers']['status-code']); // insert
+        $this->assertEquals(403, $docs[1]['headers']['status-code']); // update
```

Comment:
I think the second hit should be a 408 too right? Because we made the request and it timed out and _then_ got blocked

Reply:
I think we are ok:
403 = TIMEOUT_BLOCKED
408 = TIMEOUT
_APP_SLOW_QUERIES_MAX_HITS = 2 (when we reach 2 so get blocked)
On the second we actually got blocked so 403?